src/doc/rustc/src/platform-support/csky-unknown-linux-gnuabiv2.md

```diff
@@ -0,0 +1,33 @@
+# `csky-unknown-linux-gnuabiv2`
+
+**Tier: 3**
+
+This target supports [C-SKY](https://github.com/c-sky) v2 CPUs with `glibc`.
```

Comment:
Is it v2 for the isa or the abi? This sentence reads like it is for the isa, but the target triple suggests it is for the abi.

Reply:
fixed